docs/languages/java.md

```diff
@@ -9,9 +9,9 @@ MetaDescription: Learn about Visual Studio Code editor features (code completion
 ---
 # Java in Visual Studio Code
 
-Support for Java in Visual Studio Code is provided through a wide range of [extensions](/docs/editor/extension-marketplace.md). Combined with the power of core VS Code, these extensions give you a lightweight and performant code editor that also supports many of the most common Java development techniques.
+Support for Java in Visual Studio Code is provided through a wide range of [extensions](/docs/java/extensions.md). Combined with the power of core VS Code, these extensions give you a lightweight and performant code editor that also supports many of the most common Java development techniques.
```

Comment:
The link here is to point readers to the general VS Code extensions topic, in case they aren't familiar with how VS Code extensions are installed or work.

Reply:
Maybe we can change the paragraph to mention both
a) indicate that we have a marketplace that provides many extensions
b) mention that for Java specifically, we have the Jave extension pack

Reply:
Yeah, I think if we want to mention VS Code extensions in general, the sentence would need to be rewritten. The way it currently is worded doesn't combine well with a general link — in particular the start of the sentence, which explicitly limits its scope to "Support for Java". If you agree with a rewording, @gregvanl, I'd be happy to put up a proposal.

Reply:
Reading this section again, it does seem to be specific to Java extensions, so I'm fine with the change.